### ISSUE TITLE
Question about animated images

### ISSUE THREAD

**Issue by Rasterisko** [tsujan/FeatherNotes#296]
Hello, tsujan

First, your software is awesome! :+1: 
I want to ask if there is a way to put animated images as a gif or something else in the notes (and show the animation).

Thanks!

Edit.: oh, and I'm sorry if there is a way in more current versions, but I'm using 0.5.1.

**Comment by tsujan** [comment]
Hi,

No. FeatherNotes embeds images into its file as Base64-encoded byte arrays. Once an image is embedded, it'll be independed of external resources. A GIF image can't be embedded; only one of its frames can.

Of course, one can write a browser-like app for placing all kinds of playable videos/audios inside texts, but that's not FeatherNotes' job.

**Comment by tsujan** [comment]
> ... but I'm using 0.5.1.

Although not related to GIF, 0.5.1 is extremely old. So many things have changed since it was released. You can find a long list in [ChangeLog](https://github.com/tsujan/FeatherNotes/blob/master/ChangeLog).

**Comment by Rasterisko** [comment]
Thanks for the answer!
About the version I know its old, but I don't understand anything about programming and building apps. I've tried building a few before and had no idea what I was doing. I'm using the version available in the Linux Mint repository. Hope they update one day...

**Comment by tsujan** [comment]
> I've tried building a few before and had no idea what I was doing.

Users shouldn't need to know these things — although if they do, they could go beyond what their distros provide.

Anyway, it's always preferable *not* to install directly from the source but to make an installable package out of it first (a deb package, in your case).

> I'm using the version available in the Linux Mint repository

Strange! Mint is based on Debian, and Debian has a deb package for the latest release of FeatherNotes (https://packages.debian.org/bookworm/feathernotes). I have no idea why Mint has such an outdated version of FeatherNotes.

Please also note that installing deb packages of Debian in a non-Debian system isn't good practice and may result in crashes.

**Comment by Rasterisko** [comment]
Hello again, tsujan

Just to clarify I realesed the version 0.5.1 of Feathernotes on Linux Mint is because I'm using xfce 20.3. I downloaded the lastest version 21.1 and the version in the repository is 0.8.

About the link you send-me I couldn't install the packages. When I try to install the first one it shows a red space with nothing there:
![image](https://user-images.githubusercontent.com/95868625/220935615-b7d02630-f79f-4584-8b0c-2c15bf7ee5fb.png)

Anyway, the version 0.5.1 it has not given me any problems and for now I will continue with it. Thanks for the support!
